### ISSUE TITLE
Adding spells to action bar that is set to hide while in combat makes the bar disappear

### ISSUE THREAD

**Issue by radhaz** [zstenger93/zUI#33]
Dragging spells from the spell book to action bar 6 which is set to "hide while in combat" will cause it to vanish. I can move spells that are already on the bar without issue and /reload brings the action bar back WITH the added spell.

**Comment by zstenger93** [comment]
Are you sure, this is caused by my addon? Can you reproduce it?
The thing is currently my settings are only modifying bars 1-5. 6-8 is not modified atm.
I tried to reproduce it though in multiple ways, but I could never manage it it disappear.

Anyway at least in the process I found that bar 4-5 which are mouseovers are not working properly in combat :D
It's not a big deal (for me) but I'll check how can I fix it

**Comment by radhaz** [comment]
> @radhaz Does it only reappear with a reload or can you bring it back let's say with opening talents/spellbook which automatically shows them?

I just tested it and opening the spellbook does make the action bar reappear.

**Comment by zstenger93** [comment]
> > Are you sure, this is caused by my addon? Can you reproduce it? The thing is currently my settings are only modifying bars 1-5. 6-8 is not modified atm. I tried to reproduce it though in multiple ways, but I could never manage it it disappear.
> > Anyway at least in the process I found that bar 4-5 which are mouseovers are not working properly in combat :D It's not a big deal (for me) but I'll check how can I fix it
> 
> So first off I apologize for providing bad info as I use both action bar 5 & 6 and hide when in combat and I got them mixed up. The affected action bar is Action bar 5.
> 
> I can produce this error 100% of the time with all addons disabled except for zUI enabled. If you want I can send you my zUI SavedVariables or any other data that might be useful.

Ahh gotcha, thanks for clarifying.
I think I know what are you referring to, that was what I found out now when I tried to reproduce it :)
It is not disappearing, but the way I am doing it right now(the mouseover) is blocked in combat by Blizzard so it won't show up. It's a sort of safety measure against certain things.

Also when you open spellbook for example in combat they show up, but won't hide, which can be fixed a reopen/close after combat at least

As soon as I find a workaround I will update it!

**Comment by radhaz** [comment]
Not sure if this is related or a new issue but I'm noticing now action bar 5 is sometimes disappearing and never coming back after combat, only returning after a "reload". Opening the spellbook will make the action bar appear; however, once I close the book it goes away.

**Comment by zstenger93** [comment]
I haven't touched yet this part since then because I couldn't reproduce the problem, for now, all I have seen is what I mentioned above.

**Comment by zstenger93** [comment]
> Not sure if this is related or a new issue but I'm noticing now action bar 5 is sometimes disappearing and never coming back after combat, only returning after a "reload". Opening the spellbook will make the action bar appear; however, once I close the book it goes away.

Until this date I still couldn't replicate this exact problem, it always works fine if I add something to the bar in combat from the spellbook for example.
Also I think it's not really common/frequent thing to do so.
The mouseover bars have been improved to work better, but the mouseover part (so when you hover over the bar and it should be visible)  still won't work in combat because of blizzard limitations and I haven't found a safe way around it yet which is not blocked by them and works in the same time.

**Comment by radhaz** [comment]
> Until this date I still couldn't replicate this exact problem, it always works fine if I add something to the bar in combat from the spellbook for example. 

It's probably something on my end if it can't be replicated and no one else is having the issue, it's fine if  you would like to close the issue to clear it off the bug tracker.
